tests/integration/Microsoft.Bot.ApplicationInsights.WebApi.Tests/Microsoft.Bot.ApplicationInsights.WebApi.Tests.csproj

```
@@ -75,7 +75,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\..\libraries\integration\Microsoft.Bot.Builder.Integration.ApplicationInsights.WebApi\Microsoft.Bot.Builder.Integration.ApplicationInsights.WebApi.csproj">
-      <Project>{400bb78f-afdc-4b1c-ad85-be159b576fce}</Project>
+      <Project>{e0223463-97c6-4108-b5a9-afba16b9d5cc}</Project>
```

Change this back to the original one.
```suggestion
      <Project>{400bb78f-afdc-4b1c-ad85-be159b576fce}</Project>
```